### DISCUSSION TITLE
How should we curate `env_local_scale` for soil & what is the design?

### DISCUSSION THREAD

**Discussion by mslarae13** [microbiomedata/issues#835] [General]
FOR SOIL

Currently, `env_local_scale` for [NMDC](https://microbiomedata.github.io/berkeley-schema-fy24/env_local_scale/) has a manually curated list of permissible values in `EnvLocalScaleSoilEnum` for [the submission portal](EnvLocalScaleSoilEnum).

According to GSC, `env_local_scale` should be [multivalued](https://genomicsstandardsconsortium.github.io/mixs/0000013/). 
(Though, that's just in the description and not in the schema "Report the **entity or entities** which are in the sample or specimens local vicinity")

NMDC does not allow for this.

```
name: env_local_scale
alias: env_local_scale
domain_of:
- Biosample
range: ControlledIdentifiedTermValue
multivalued: false
```

In this discussion I'd like to determine
1. The rules / guidance NMDC will use to curate the env_local_scale permissible values for soil in the NMDC submission portal
     - What branches? How do we query OAK for this?
2. Should this term be multivalued?
    - allow for multi-select in the submission portal like we do for analysis_type
3. Does each extension need a unique enum? How much overlap is possible in each extension?

@turbomam @cmungall @lamccue @Natalie-Winans

**Comment by turbomam** [comment]
As I mentioned in
- https://github.com/microbiomedata/issues/discussions/837

I think decisions about multi-valued are important but outside the scope of a value set squad.

**Comment by mslarae13** [comment]
Decision was made in last weeks meeting for an initial subset creation. See [ADR](https://github.com/microbiomedata/issues/pull/881), but in summary

NMDC General Query
- `env_local_scale` will be composed of terms that branch from material entity [BFO:0000040], excluding biome [ENVO:00000428] & environmental material [ENVO:00010483]
  - Additional narrowing will occur to remove specific branches. 

For **soil** environment (MIxS Extension)
- `env_local_scale` ????? TBC

This will be refined & updates after evaluation, as aligned with @turbomam 's comment in a discussion around `env_medium` . https://github.com/microbiomedata/issues/discussions/837#discussioncomment-10388452
